07LessonSeven/homework/css/style.css

```diff
@@ -41,8 +41,8 @@ a, p {
   border: 2px solid white;
   padding: 10px;
 }
-.clear-fix:after {
+/*.clear-fix:after {
```

Comment:
i believe you need this. Take a look to the element `<section class="signup">...</section` and add a background color. You won't see the color. 
Or try adding something after that element, and you will noticed something it's not right